engine/servermaster/server.go

```diff
@@ -518,10 +512,12 @@ func (s *Server) registerMetaStore() error {
 	if err != nil {
 		return err
 	}
-	if s.userMetaKVClient, err = meta.NewKVClient(cfg.UserMetaConf); err != nil {
+	userMetaKVClient, err := meta.NewClientConn(cfg.UserMetaConf)
+	if err != nil {
 		log.Error("connect to user metastore fail", zap.Any("config", cfg.UserMetaConf), zap.Error(err))
 		return err
 	}
+	userMetaKVClient.Close()
```

Comment:
Is this used to test user meta store is accessible?

Reply:
Yes.